drivers/spi/spi_ll_stm32.c

```diff
@@ -451,11 +634,75 @@ static int transceive(struct device *dev,
 	return ret;
 }
 
+#ifdef CONFIG_SPI_STM32_DMA
+static int transceive_dma(struct device *dev,
+		      const struct spi_config *config,
+		      const struct spi_buf_set *tx_bufs,
+		      const struct spi_buf_set *rx_bufs,
+		      bool asynchronous, struct k_poll_signal *signal)
+{
+	const struct spi_stm32_config *cfg = DEV_CFG(dev);
+	struct spi_stm32_data *data = DEV_DATA(dev);
+	SPI_TypeDef *spi = cfg->spi;
+	int ret;
+
+	if (!tx_bufs) {
+		return 0;
+	}
+	if (!rx_bufs) {
+		return 0;
+	}
```

Comment:
Using `spi_read()` and `spi_write()` will return here.
Is only `spi_transceive()` supported?